### PR TITLE
added CSV and data example

### PR DESCRIPTION
**Pull request by mgartner**
@pon @elnaz @leore @robinjoseph08 @russelltaylor05

### CODE REVIEW

File: examples/csv_postcards/create_postcards.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#$:.unshift File.expand_path("../lib", File.dirname(__FILE__))
```
Comment (robinjoseph08) [inline examples/csv_postcards/create_postcards.rb]: will this still update the load path even though it's commented?

File: examples/csv_postcards/postcard_back.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    padding: .4in .6in 0 .4in;
+  }
+</style>
+</head>
+<body>
+  <div id="left-box">
+    Hey {{name}},<br><br>
+    Your {{car}} is due for its {{mileage}} mile service. Schedule an appointment today!<br><br>
+    Make sure to bring this card to receive great savings!<br><br>
+  </div>
```
Comment (robinjoseph08) [inline examples/csv_postcards/postcard_back.html]: missing `</body>`

File: examples/csv_postcards/create_postcards.rb (hunk 1 above)
Comment (mgartner) [inline examples/csv_postcards/create_postcards.rb]: No I need to take that line out good catch

File: examples/csv_postcards/postcard_back.html (hunk 2 above)
Comment (mgartner) [inline examples/csv_postcards/postcard_back.html]: fixed

**Comment by pon** [comment]
LGTM

**Comment by robinjoseph08** [comment]
LGTM

**Pull request merged**